### PR TITLE
Desktop screen

### PR DESCRIPTION
**Pull request by norbiboy1**
I have organized the layout of the main.html to be responsive for desktop screens.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -933,3 +479,16 @@ hr {
   margin-top: 0;
   margin-bottom: 10px;
 }
+
+/* This is for the nav tag */
+@media (max-width: 768px) {
+  .hamburger-icon {
+    display: inline;
+    color: #6070ff;
+  }
```
Comment (descholar-ceo) [inline main.css:487]: - [ ] The Desktop version is still showing the mobile version you did earlier, kindly, make sure that it follows this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Comment (descholar-ceo) [reply main.css:487]: As a hint, you will need these breakpoints: `@media screen and (min-width: 1080px) {}` and `@media screen and (min-width: 750px) {}`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @norbiboy1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @norbiboy1, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**